raster-foundry/raster-foundry
# Make tool permissions from tools instead of datasources

## Overview

This PR fixes the migration to add public object access control rules to make tools (templates)
get permissions from the tools table instead of the datasources table. It also adds indices to the access
control rules table that will prevent us from being miserable very soon.

### Checklist


## Testing Instructions

 * reload development data (or run the line from the change to the migration manually)
 * browse templates
 * observe that you can see templates now

Helps with azavea/raster-foundry-platform#346
